### PR TITLE
Update simultaneous.md

### PR DESCRIPTION
**Pull request by ksudoh**
Add descriptions of the Simultaneous task

This version basically follows 2023

### CODE REVIEW

File: _pages/2024/shared-tasks/simultaneous.md [hunk 1]
```diff
@@ -13,40 +13,172 @@ if you want to have a line break either put a blank line in between the text or
...
-<!-- Description of metrics used for evaluation, what the official ranking is based on, links to evaluation scripts -->
+The evaluation implementation will use the latest [SimulEval](https://github.com/facebookresearch/SimulEval) toolkit.
+We will use the new [remote evaluation](https://simuleval.readthedocs.io/en/latest/tutorials/remote_evaluation.html) mode.
+
+### Metrics
+The system's performance will be evaluated in two ways:
+
+- Translation quality metrics:
+  - BLEU
+  - BLASER
```
Comment (xutaima) [inline _pages/2024/shared-tasks/simultaneous.md]: For simplicity, maybe we can remove the BLASER this year.

File: _pages/2024/shared-tasks/simultaneous.md [hunk 2]
```diff
@@ -13,40 +13,172 @@ if you want to have a line break either put a blank line in between the text or
...
+Based on analysis on the quality-latency tradeoffs for the baseline systems,
+for all three languages, the thresholds are set as follows:
+
+- Speech-to-Text Translation: 2 seconds (Average Lagging)
+- Speech-to-Speech Translation: 2.5 seconds (Starting Offset)
 
 ## Organizers
 
-<!-- List of organizers' names and affiliations -->
+- Katsuhito Sudoh (NAIST)
```
Comment (xutaima) [inline _pages/2024/shared-tasks/simultaneous.md:159]: From the discussion yesterday, maybe we can stick to a shorter list for now. I am following up with Shinji on whether he is interested in organizing this year.

```
(to be updated)
- Katsuhito Sudoh (NAIST)
- Ondřej Bojar (Charles University)
- Xutai Ma (Meta)
- Juan Pino (Meta)
```

**Review by xutaima — COMMENTED** [review]
Thank you so much! It looks good for me. Just left some minor comments

**Review by xutaima — APPROVED** [review]

File: _pages/2024/shared-tasks/simultaneous.md [hunk 3]
```diff
@@ -13,40 +13,172 @@ if you want to have a line break either put a blank line in between the text or
...
-<!-- Description of expected submission format and submission instructions -->
+### Format
+
+Participants can have multiple submissions,
+but can only have one submission for one track (`s2t`/`s2s`), one language (`de`/`zh`/`ja`).
+
+One submission should contain the following items:
+
+- **A built docker image file**, named `image.tar`.
+- The output logs of the system on **MuST-C v2.0 tst-COMMON** sets, named `results`.
```
Comment (jmp84) [inline _pages/2024/shared-tasks/simultaneous.md:62]: It is important to use v2 as the test set is slightly different from the first version.

File: _pages/2024/shared-tasks/simultaneous.md [hunk 4]
```diff
@@ -13,40 +13,172 @@ if you want to have a line break either put a blank line in between the text or
...
+- A **readme** file containing information needed to run the system, named `readme`.
+
+The submission should be compressed as a `tar.gz` file,
+and uploaded to the [dropbox](https://www.dropbox.com/scl/fo/g623o0jyndqahcblmz5gn/h?dl=0&rlkey=6e441oy6uhn64ldqxmbomn7c7). For data security reason, you might need to request the access before the uploading the file.
```
Comment (jmp84) [inline _pages/2024/shared-tasks/simultaneous.md]: LINK TBD

File: _pages/2024/shared-tasks/simultaneous.md [hunk 5]
```diff
@@ -13,40 +13,172 @@ if you want to have a line break either put a blank line in between the text or
...
+
+```
+tar -xf meta_text_de_low.tar.gz -C meta_text_de_low
+> ls meta_text_de_low
+image.tar results readme
+```
+
+### Docker Image
+
+The docker image should already be built and able to run in a controlled environment, specifically an AWS [ap3.2xlarge](https://aws.amazon.com/ec2/instance-types/p3/) instance. Please make sure that the **docker image alone** is enough to run the system. All dependencies, such as checkpoints and dictionaries should be included inside the image.
```
Comment (jmp84) [inline _pages/2024/shared-tasks/simultaneous.md]: Should we revisit the AWS instance?

File: _pages/2024/shared-tasks/simultaneous.md (hunk 1 above)
Comment (jmp84) [reply _pages/2024/shared-tasks/simultaneous.md]: ok. for speech to speech, specify ASR-BLEU ?

File: _pages/2024/shared-tasks/simultaneous.md [hunk 6]
```diff
@@ -13,40 +13,172 @@ if you want to have a line break either put a blank line in between the text or
...
+The evaluation implementation will use the latest [SimulEval](https://github.com/facebookresearch/SimulEval) toolkit.
+We will use the new [remote evaluation](https://simuleval.readthedocs.io/en/latest/tutorials/remote_evaluation.html) mode.
+
+### Metrics
+The system's performance will be evaluated in two ways:
+
+- Translation quality metrics:
+  - BLEU
+  - BLASER
+- Translation latency:
```
Comment (jmp84) [inline _pages/2024/shared-tasks/simultaneous.md:131]: Is there any chance to simply and use a smaller number of latency metrics?

**Comment by ksudoh** [comment]
Thank you for your suggestions!
I basically followed them except for the problem of many latency metrics :-(
Can I eliminate AP, and some of the AL variants?

**Comment by xutaima** [comment]
> Thank you for your suggestions! I basically followed them except for the problem of many latency metrics :-( Can I eliminate AP, and some of the AL variants?

Hi @ksudoh thanks for bring this up. Can we delete AP and DAL? LAAL seems pretty popular these days and maybe we can keep it.

**Comment by ksudoh** [comment]
Thank you for your comments!

**Pull request merged**